src/cljc/rems/text.cljc

```diff
@@ -63,6 +63,20 @@
 (defn localize-state [state]
   (text (get states state :t.applications.states/unknown)))
 
+(def ^:private todos
+  {:new-application :t.applications.todos/new-application
+   :no-pending-requests :t.applications.todos/no-pending-requests
+   :resubmitted-application :t.applications.todos/resubmitted-application
+   :waiting-for-decision :t.applications.todos/waiting-for-decision
+   :waiting-for-review :t.applications.todos/waiting-for-review
+   :waiting-for-your-decision :t.applications.todos/waiting-for-your-decision
+   :waiting-for-your-review :t.applications.todos/waiting-for-your-review})
+
+(defn localize-todo [todo]
+  (if (nil? todo)
```

Comment:
Hmm, is this check needed anymore? If for some reason a nil value ends up as far as here, the "unknown" text would probably suffice.

Reply:
It's nil before submitting and after approve/reject/close. Those states are not shown in the todo table, but they will be found by the application search in the other tables.

Reply:
Ok right. Is there any known case in which unknown todo text is showed? Maybe that could be replaced by an empty string to make the function more simple and get rid of the unneeded translations?

Reply:
Or actually, never mind, I think an explicit nil check is good to have here to make the logic clear.

Reply:
The unknown text would be shown if the backend returns a keyword for which there is no translation, i.e. somebody forgot to update the frontend.

Reply:
Right, I think it's fine. Thanks.